src/main/java/org/mockito/internal/configuration/MockAnnotationProcessor.java

```diff
@@ -45,10 +46,12 @@ public static Object processAnnotationForMock(
         if (annotation.stubOnly()) {
             mockSettings.stubOnly();
         }
-        mockSettings.strictness(annotation.strictness());
         if (annotation.lenient()) {
             mockSettings.lenient();
         }
+        if (annotation.strictness() != Mock.Strictness.NOT_SET) {
+            mockSettings.strictness(annotation.strictness().outer());
```

Comment:
Rather than defining an `outer` method, can we instead use the dynamic enum valueOf here?

Reply:
sure